src/tox_travis/after.py

```diff
@@ -81,10 +80,10 @@ def travis_after():
     print('All required jobs were successful.')
 
 
-def after_config_matches():
+def after_config_matches(config):
     """Determine if this job should wait for the others."""
-    config = py.iniconfig.IniConfig('tox.ini')
-    section = config.sections.get('travis:after', {})
+    tox_ini = get_iniconfig(config)
+    section = tox_ini.sections.get('travis:after', {})
```

Comment:
The only thing I don't like about this is the name `tox_ini`. It's pretty clear what it is, but we also know that `tox_ini` no longer needs to be anything to do with `tox.ini`, it could just as easily be `setup.cfg`, and then `setup_cfg` would seem more appropriate. I wish we didn't need `config` for the special object from Tox.

Perhaps we could just use `iniconfig` instead as the variable name? Also, this this case I don't think we need to have the intermediate variable anyway.